README.md

```diff
@@ -0,0 +1,64 @@
+# Common Code of the Main Telescope Rotator and Hexapod Controllers
```

Comment:
This title should be modified to M2. It is not for the rotator and hexapod.

Reply:
done